static/app/components/events/profileEventEvidence.tsx

```diff
@@ -20,22 +22,22 @@ export const ProfileEventEvidence = ({event, projectSlug}: ProfileEvidenceProps)
           {
             subject: 'Transaction Name',
             key: 'Transaction Name',
-            value: (
-              <pre>
-                <Link
-                  to={getTransactionDetailsUrl(
-                    organization.slug,
-                    generateEventSlug({
-                      id: evidenceData.transactionId,
-                      project: projectSlug,
-                    }),
-                    undefined,
-                    {referrer: 'issue'}
-                  )}
-                >
-                  {evidenceData.transactionName}
-                </Link>
-              </pre>
+            value: evidenceData.transactionName,
+            actionButton: (
+              <Button
+                size="xs"
+                to={getTransactionDetailsUrl(
+                  organization.slug,
+                  generateEventSlug({
+                    id: evidenceData.transactionId,
+                    project: projectSlug,
+                  }),
+                  undefined,
+                  {referrer: 'issue'}
+                )}
+              >
+                {t('View Transaction')}
```

Comment:
@malwilley Do we even have this for errors?

Reply:
We don't, errors don't have an evidence section at all

Reply:
@malwilley Yep...it's something I noticed using the product today. It's too hard to get from an error to the transaction overview. It seems like this should be standardized across all issue types as each of them can link back to a transaction.